### DISCUSSION TITLE
GS-T3E v2 Zigbee

### DISCUSSION THREAD

**Discussion by stefan242** [HASwitchPlate/openHASP#578] [Hardware]
I just installed OpenHASP 0.7rc on my new GS-T3E, which supported Zigbee with the original firmware. I have added the display to zigbee2mqtt and it is detected as (another) TS0601, but it is not supported:
Zigbee-Model: TS0601
Zigbee-manufacturer: _TZE200_mua6ucdj

With the Original firmware I did not activate Wifi, but with OpenHASP installed, everything works perfectly.
Just one question: Can I use the Zigbee chip on the GS-T3E to connect it to my Zigbee stick and maybe control it from there?
Since Wifi works, this would just be nice to have.
BTW: I have another identical display still with its Original firmware, so if someone needs the display to play with it, I could lend (!) it to him/her.

**Comment by stefan242** [comment]
Is this the "ZS3L Tuya chip on PCB (unused)", for which you wrote: "openHASP does not support the proprietary Tuya chip, but you can still flash the firmware and use the other GS-T3E features just fine."
Then sorry for that!

**Reply by fvanroie** [reply]
No effort has been made to communicate with the Tuya chip or implement its protocol. openHASP works based on MQTT protocol, so the chip just sits there and is not being used for anything with openHASP firmware...

**Reply by yabdali** [reply]
Sorry to jump in the middle of this discussion. I have 2 units of the T3E V2 which I haven't used yet as I am waiting for my Tuya gateway order to arrive. I am testing various Zigbee switches as I am in the process of finishing my new house within 2-3 months timeline. I needed something like the T3E to be installed by the bedside so I can control the lights at the room entrance and the ambient lights. Was considering Sonoff BS Panel Pro but found that it is still work in progress and lacking some features besides not recognizing a wide type of devices. 
If I may ask, how do you see the OpenHASP experience compared to the original firmware? I can see from the discussion that the Zigbee chip will not be utilized when OpenHASP is installed. How about the original firmware, was the WIFI working and recognized by Home Assistant? If so, could it be used with HA using WIFI without flashing OpenHASP?

One final question, regarding the installation of OpenHASP, is the this the right procedure for putting the device into bootload mode? Connecting each pinpoint and plugging the other end of the USB cable to the the PC to get into bootload mode then removing the wire connecting GND with GPIO0 and proceeding with the rest of the steps?
Thanks and sorry for the lengthy set of questions...

![image](https://github.com/HASwitchPlate/openHASP/assets/30695308/23e702dc-76c6-4e5d-b19d-4c64371568d2)

**Reply by stefan242** [reply]
Sorry for the delay. I don't use HA, so I have no experience with it.
AFAIR, you can leave the wire connecting GND with GPIO0. No harm is done. But I used the small "key" button to activate the bootload mode.

> How about the original firmware, was the WIFI working

I didn't activate WIFI, because I didn't want Tuya to get this data. So I just was able to switch the three relays locally on and off, but could not establish a connection with zigbee2mqtt. So I switched to OpenHASP.
The display is great compared to the Lanbon 8, which I already use as a switch, but I love the mood light of the Lanbon and use it to display the state of my windows, while opening the front door. (green = no display, all windows and doors cloded | yellow =  show display with not critical windows open | red = show display with critical windows and doors still open)

**Reply by yabdali** [reply]
Thank you. I am testing two devices of this model with the original firmware before I make a decision on what I am going to do with it.

**Comment by stefan242** [comment]
If no one needs the original display with the stock firmware, I will flash it with OpenHASP like the other one.

**Comment by djkennny** [comment]
hi, does anyone have a backup of the original fw?

**Reply by stefan242** [reply]
No, I have flashed both of my displays.
